packages/fluentui/react-northstar/src/components/Input/Input.tsx

```diff
@@ -150,9 +161,14 @@ const Input = compose<'input', InputProps, InputStylesProps, {}, {}>(
       error,
       errorIndicator,
       showSuccessIndicator,
+      label,
+      labelPosition,
     } = props;
+
     const inputRef = React.useRef<HTMLInputElement>();
 
+    const { labelId } = React.useContext(FormFieldBaseContext);
```

Comment:
What was the reason for extracting only this in the context, versus propagating it via props?